app/controllers/medicaid/citizens_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class CitizensController < Medicaid::ManyMemberStepsController
+    private
+
+    def skip?
+      everyone_a_citizen?
+    end
+
+    def member_attrs
+      %i[citizen]
+    end
+
+    def everyone_a_citizen?
+      current_application.citizen?
```

Comment:
while are are here, does it make sense to rename the `citizen` column to `everyone_a_citizen` on medicaid apps? that's the name it has on on snap apps and we've been trying to standardize as we go when possible